README.md

```diff
@@ -16,18 +16,15 @@ Wakeword voice service for MALOS. The last version support:
 
 ### Raspbian Dependencies 
 
-Before, please install FPGA and MCU drivers on your RaspberryPi3 and perform device reboot. 
+Before, please install MALOS on your RaspberryPi3 and perform device reboot. For more details: [Getting Started Guide](https://github.com/matrix-io/matrix-creator-quickstart/wiki/2.-Getting-Started)
```

Comment:
For now please restore installation flow, because we need drivers and then malos-wakeword